### DISCUSSION TITLE
forza-horizon-5-creative-accolades/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-personal#8] [Comments]
# forza-horizon-5-creative-accolades/

Forza Horizon 5 is easily my favourite game of 2021/22, and it contains a comprehensive “accolade” (achievements) system. Earning these accolades improves your global hall of fame ranking, as well as various rewards along the way. However, the Creative category contains many accolades that are seemingly impossible to complete. Here’s how I got them, since I couldn’t find any guides!

https://jakelee.co.uk/forza-horizon-5-creative-accolades/

**Comment by christianopaets** [comment]
If somebody is still doing this, I found that during downloading tunes, if you choose a car that is different from tuned one, you can skip the step of selecting "Download and save", after clicking Enter/A button it will donwload automatically.
